packages/client/src/_types/global/GlobalTypes.ts

```diff
@@ -1,9 +1,8 @@
+import type { GameManager } from "@backend/GameLogic/GameManager";
+import type { GameUIManager } from "@backend/GameLogic/GameUIManager";
```

Comment:
Importing the correct exported classes, and also from the aliased `@backed` package.